apps/prairielearn/elements/pl-image-capture/pl-image-capture.js

```diff
@@ -810,14 +963,22 @@ const MAX_ZOOM_SCALE = 5;
       }
     }
 
-    showCropRotateButton() {
+    /**
+     * Set if the crop rotate button is shown or not.
+     *
+     * @param {boolean} show If true, shows the crop rotate button. Otherwise, hides it.
+     */
+    setShowCropRotateButton(show) {
       const cropRotateButton = this.imageCaptureDiv.querySelector('.js-crop-rotate-button');
 
       this.ensureElementsExist({
         cropRotateButton,
       });
-
-      cropRotateButton.classList.remove('d-none');
+      if (show) {
+        cropRotateButton.classList.remove('d-none');
+      } else {
+        cropRotateButton.classList.add('d-none');
+      }
```

Comment:
Nit, this should be equivalent and is a little more concise:

```suggestion
      cropRotateButton.classList.toggle('d-none', !show);
```